index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="css/style.css" />
+	<script src="https://kit.fontawesome.com/102965b95f.js" crossorigin="anonymous"></script>
+	<title>Document</title>
+</head>
+
+<body>
+	<header class="header">
+		<nav class="bar">
+			<ul class="bar__cont">
+				<li class="bar__item"> <a class="bar__item" href="#"><i class="fa fa-twitter"></i> </a></li>
+				<li class="bar__item"><a class="bar__item" href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="bar__item"><a class="bar__item" href="#">English</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">My Page</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">Logout</a></li>
+			</ul>
```

Comment:
- [ ] There is horizontal scroll problem in your index.html that doesn't looks good. Please fix this.


![Screenshot from 2021-07-02 13-30-29](https://user-images.githubusercontent.com/22434660/124242191-1ad9ce80-db3a-11eb-8e2a-926dbedc9b8c.png)

Reply:
- [ ] The horizontal scroll bar is still there, kindly fix it.

Reply:
@tenmarie thanks for the feedback, this was more about the big white margin rather than the scrollbar. I will remove the overflow though.